### ISSUE TITLE
Q: EPG?

### ISSUE THREAD

**Issue by alexfd** [dexterke/plugin.video.digi.ro-online#7]
Hi @dexterke,
Great job with this addon. Works great for me.
I've seen some references in the code regarding EPG, but I cannot find it at all in Kodi. Where should it be displayed?

**Comment by dexterke** [comment]
Hi @alexfd 
Thank you for your comment.
There is no EPG for this plugin, I have no idea if it's even possible.

**Comment by alexfd** [comment]
Sorry, I now realized I was really loose in my definition of EPG, my bad. I actually was imagining I was talking about a list of currently playing titles, that would update every x minutes.  
This would be super useful if it was in the channel index, under each channel.  

Something similar to  

   """Extract 'now-playing' info from HTML."""
                nowPlaying_Info = " - "
                if osdInfo_Enabled == 'true':
                    txt = str((re.compile('<h2 class="section-title-alt" id="title">(.+?)<\/h2>').findall(html_text))[0])
                    write2file(log_File, 'processHTML nowPlaying txt: ' + txt, 'a')
                    nowPlaying_Info = html_parser.unescape(txt).replace("&period;", ".").replace("&colon;", ":").replace("&amp;", "&").replace("&commat;", "@")  

which I believe is the little popup displayed after pressing Play.  
Was looking through their source and found this could be achieved through either grabbing the title by filtering the <h5> tags from https://www.digionline.ro/filme OR from a JSON request like https://www.digionline.ro/epg-xhr?channelId=44, though this would probably mean a bit too many requests.  
Would you be willing to implement this? I'd do a PR but my python skills are == 0. Only thing I know is bash "programming".

**Comment by dexterke** [comment]
Yes, toggle `OSD Info Popup` control in the settings tab to see what is playing when loading the media files.
I was planning on adding a poller thread to run in the background after the video would have started and keep updating `nowPlaying_Info` as long as the media link is being played, similar to  https://github.com/dexterke/plugin.video.digi.hu-online/blob/master/default.py 

I will get to it eventually, not sure when. Currently I'm tied up in trying to understand why this plugin is broken on Android ( issue #4 )

**Comment by alexfd** [comment]
Yup, `OSD Info Popup` works.
Here's what I was referring to:
![screenshot002](https://user-images.githubusercontent.com/1685977/73472177-825a8000-4382-11ea-96ea-0d83be5c02ce.png)
Similar to what my PVR client shows, but without the description on the right. Just the current playing title would be super helpful.
![screenshot000](https://user-images.githubusercontent.com/1685977/73472239-a0c07b80-4382-11ea-8538-eaa2db56fb40.png)
